doc/using_tf.rst

```diff
@@ -133,6 +133,26 @@ In your training script the channels will be stored in environment variables ``S
    ``output_path``.
 
 
+Use third-party libraries
+-------------------------
+
+If there are other packages you want to use with your script, you can use a ``requirements.txt`` to install other dependencies at runtime.
+
+Support for installing packages using ``requirements.txt`` varies with Tensorflow versions:
+
+- For Tensorflow 1.15.2 with Python 3.7 or newer, Tensorflow 2.2 or newer, and Script Mode without Horovod in any version:
+    - Include a ``requirements.txt`` file in the same directory as your training script.
+    - You must specify this directory using the ``source_dir`` argument when creating a Tensorflow estimator.
+- For older versions of Tensorflow using Script Mode with Horovod:
+    - Write a shell script for your entry point that first calls ``pip install -r requirements.txt``, then runs your training script.
+- For older versions of Tensorflow using Framework Mode:
+    - Specify the path to your ``requirements.txt`` file using the ``requirements_file`` argument.
+
```

Comment:
The other two frameworks (perhaps erroneously) included how to use requirements.txt + serving in this part of the documentation. I'd recommend making sure we add it somewhere, though it doesn't have to be in this section (since it's more specific to training)

Reply:
Looks like TF serving containers depend on `requirements.txt` being placed in the `code` directory: https://github.com/aws/sagemaker-tensorflow-serving-container/blob/b7b79b467dcf3113bc49154178d6239a3287a2ea/docker/build_artifacts/sagemaker/serve.py#L33

I'll add a line to the docs indicating this.